### PR TITLE
allow panotpes zoo id formats

### PR DESCRIPTION
**Pull request by camallen**
events controller is rejecting new user activities on old projects

### CODE REVIEW

File: app/schemas/event_create_schema.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ class EventCreateSchema < JsonSchema
 
     property "zooniverse_user_id" do
       type "string", "integer"
-      pattern "^[0-9]*$"
+      pattern "^.+[0-9]*$"
```
Comment (edpaget) [inline app/schemas/event_create_schema.rb]: Do you want to make this pattern more explicity? as in "^(panoptes-|)[0-9]+$"

Comment (camallen) [inline app/schemas/event_create_schema.rb]: i could i just don't want to have to redo. It'll currently fail on finding the user and not just reject the message outright before even attempting to process.

Comment (edpaget) [inline app/schemas/event_create_schema.rb]: Sure. Not like anyone but us will be using this endpoint.

**Comment by edpaget** [comment]
:shipit:

**Pull request merged**